Storybook build broken for security_solution and security_solution_packages

The storybook for the security_solution and security_solution_packages started failing ([example run](https://buildkite.com/elastic/kibana-on-merge/builds/59475#019483d6-1d50-403c-aba0-5ab5fb9c612a)).

```

info => Manager built (5.02 s)
--
  | ERR! => Failed to build the preview
  | ERR! ./src/platform/packages/shared/kbn-discover-utils/src/__mocks__/es_hits.ts 72:17
  | ERR! Module parse failed: Identifier directly after number (72:17)
  | ERR! File was processed with these loaders:
  | ERR!  * ./node_modules/babel-loader/lib/index.js
  | ERR! You may need an additional loader to handle the result of these loaders.
  | ERR! \| });
  | ERR! \| var baseDate = new Date('2024-01-1').getTime();
  | ERR! > var dateInc = 100_000_000;
  | ERR! \| var generateFieldValue = function (field, index) {
  | ERR! \|   switch (field.type) {
  | ERR!  @ ./src/platform/packages/shared/kbn-discover-utils/src/__mocks__/index.ts 11:0-26 11:0-26
  | ERR!  @ ./src/platform/plugins/shared/discover/public/__mocks__/data_view_esql.ts
  | ERR!  @ ./src/platform/plugins/shared/discover/public/__mocks__/saved_search.ts
  | ERR!  @ ./src/platform/plugins/shared/discover/public/__mocks__/discover_state.mock.ts
  | ERR!  @ ./src/platform/plugins/shared/discover/public/mocks.tsx
  | ERR!  @ ./x-pack/solutions/security/plugins/security_solution/public/common/lib/kibana/kibana_react.mock.ts
  | ERR!  @ ./x-pack/solutions/security/plugins/security_solution/public/common/mock/create_store.ts
  | ERR!  @ ./x-pack/solutions/security/plugins/security_solution/public/common/mock/storybook_providers.tsx
  | ERR!  @ ./x-pack/solutions/security/plugins/security_solution/public/flyout/entity_details/host_right/content.stories.tsx
  | ERR!  @ ./x-pack/solutions/security/plugins/security_solution sync ^\.(?:(?:^\|\/\|(?:(?:(?!(?:^\|\/)\.).)*?)\/)(?!\.)(?=.)[^/]*?\.stories\.tsx)$
  | ERR!  @ ./generated-stories-entry.js
  | ERR!  @ multi ./node_modules/@storybook/core-client/dist/esm/globals/polyfills.js ./node_modules/@storybook/core-client/dist/esm/globals/globals.js ./storybook-init-framework-entry.js ./node_modules/@storybook/react/dist/esm/client/docs/config-generated-config-entry.js ./node_modules/@storybook/react/dist/esm/client/preview/config-generated-config-entry.js ./node_modules/@storybook/addon-a11y/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-docs/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-actions/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-backgrounds/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-measure/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-outline/preview.js-generated-config-entry.js ./packages/kbn-storybook/src/lib/decorators.tsx-generated-config-entry.js ./generated-stories-entry.js
  | ERR! ./node_modules/@testing-library/react/dist/index.js 23:51
  | ERR! Module parse failed: Unexpected token (23:51)
  | ERR! You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
  | ERR! \| // if you don't like this then either import the `pure` module
  | ERR! \| // or set the RTL_SKIP_AUTO_CLEANUP env variable to 'true'.
  | ERR! > if (typeof process === 'undefined' \|\| !process.env?.RTL_SKIP_AUTO_CLEANUP) {
  | ERR! \|   // ignore teardown() in code coverage because Jest does not support it
  | ERR! \|   /* istanbul ignore else */
```

This was temporarily skipped at https://github.com/elastic/kibana/pull/207225

This was the error surfaced in backport builds

```bash
➜ kibana (backport/8.x/pr-195148) ✗ node scripts/storybook --site security_solution
info => Cleaning outputDir: built_assets/storybook/security_solution
info => Loading presets
info => Building manager..
info => Manager built (270 ms)
info => Building preview..
info Addon-docs: using MDX3
info => Using implicit CSS loaders
info => [@storybook/addon-styling-webpack] Applying custom Storybook webpack configuration styling.
info => [@storybook/addon-styling-webpack] Replacing Storybook's webpack rules for styles with given rules.
info => Using default Webpack5 setup
10% building 0/1 entries 0/0 dependencies 0/0 modulesinfo => Copying static files: bazel-bin/src/platform/packages/private/kbn-ui-shared-deps-npm/shared_built_assets at built_assets/storybook/security_solution
46% building 0/1 entries 168/176 dependencies 55/82 modulesWARN No story files found for the specified pattern: x-pack/solutions/security/plugins/security_solution/**/*.mdx
=> Failed to build the preview
ERROR UNHANDLED ERROR
ERROR SB_BUILDER-WEBPACK5_0002 (WebpackInvocationError): Module not found: Error: Can't resolve 'react-dom/client' in '/home/brad/kibana/node_modules/@testing-library/react/dist/@testing-library'
          at /home/brad/kibana/node_modules/@storybook/builder-webpack5/dist/index.js:1:24959
          at /home/brad/kibana/node_modules/webpack/lib/HookWebpackError.js:67:2
          at Hook.eval [as callAsync] (eval at create (/home/brad/kibana/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:11:1)
          at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/home/brad/kibana/node_modules/tapable/lib/Hook.js:18:14)
          at Cache.shutdown (/home/brad/kibana/node_modules/webpack/lib/Cache.js:154:23)
          at /home/brad/kibana/node_modules/webpack/lib/Compiler.js:1379:15
          at Hook.eval [as callAsync] (eval at create (/home/brad/kibana/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
          at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/home/brad/kibana/node_modules/tapable/lib/Hook.js:18:14)
          at Compiler.close (/home/brad/kibana/node_modules/webpack/lib/Compiler.js:1372:23)
          at /home/brad/kibana/node_modules/@storybook/builder-webpack5/dist/index.js:1:24944
          at finalCallback (/home/brad/kibana/node_modules/webpack/lib/Compiler.js:500:32)
          at onCompiled (/home/brad/kibana/node_modules/webpack/lib/Compiler.js:514:20)
          at /home/brad/kibana/node_modules/webpack/lib/Compiler.js:1327:21
          at eval (eval at create (/home/brad/kibana/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:20:1)
          at /home/brad/kibana/node_modules/webpack/lib/EntryPlugin.js:47:5
          at /home/brad/kibana/node_modules/webpack/lib/Compilation.js:2303:13
          at /home/brad/kibana/node_modules/webpack/lib/Compilation.js:2192:6
          at /home/brad/kibana/node_modules/webpack/lib/Compilation.js:2055:13
          at /home/brad/kibana/node_modules/webpack/lib/util/AsyncQueue.js:389:5
          at Hook.eval [as callAsync] (eval at create (/home/brad/kibana/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
          at AsyncQueue._handleResult (/home/brad/kibana/node_modules/webpack/lib/util/AsyncQueue.js:359:21)
          at /home/brad/kibana/node_modules/webpack/lib/util/AsyncQueue.js:342:11
          at onTransitiveTasksFinished (/home/brad/kibana/node_modules/webpack/lib/Compilation.js:1562:20)
          at /home/brad/kibana/node_modules/webpack/lib/Compilation.js:1548:7
          at /home/brad/kibana/node_modules/webpack/lib/Compilation.js:2055:13
          at /home/brad/kibana/node_modules/webpack/lib/util/AsyncQueue.js:389:5
          at Hook.eval [as callAsync] (eval at create (/home/brad/kibana/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
          at AsyncQueue._handleResult (/home/brad/kibana/node_modules/webpack/lib/util/AsyncQueue.js:359:21)
          at /home/brad/kibana/node_modules/webpack/lib/util/AsyncQueue.js:342:11
          at onTransitiveTasksFinished (/home/brad/kibana/node_modules/webpack/lib/Compilation.js:1562:20)
          at /home/brad/kibana/node_modules/webpack/lib/Compilation.js:1548:7
          at /home/brad/kibana/node_modules/webpack/lib/Compilation.js:2055:13
          at /home/brad/kibana/node_modules/webpack/lib/util/AsyncQueue.js:381:6
          at processTicksAndRejections (node:internal/process/task_queues:77:11)
```

--- a/src/dev/storybook/aliases.ts
+++ b/src/dev/storybook/aliases.ts
@@ -60,7 +60,7 @@ export const storybookAliases = {
   random_sampling: 'x-pack/platform/packages/private/kbn-random-sampling/.storybook',
   ['response-ops']: 'src/platform/packages/shared/response-ops/.storybook',
   security_solution: 'x-pack/solutions/security/plugins/security_solution/.storybook',
-  // security_solution_packages: 'x-pack/solutions/security/packages/storybook/config',
+  security_solution_packages: 'x-pack/solutions/security/packages/storybook/config',
   serverless: 'src/platform/packages/shared/serverless/storybook/config',
   shared_ux: 'src/platform/packages/private/shared-ux/storybook/config',
   streams_app: 'x-pack/platform/plugins/shared/streams_app/.storybook',

